assets/css/amp-default.css

```diff
@@ -12,4 +12,12 @@ amp-fit-text h4,
 amp-fit-text h5,
 amp-fit-text h6 {
 	font-size: inherit;
-}
\ No newline at end of file
+}
+
+#amp-image-lightbox div div > :first-child {
+	object-fit: contain;
+}
+
+.entry__content .wp-block-gallery[class*="columns-"] figure {
+	width: 100%;
```

Comment:
I think this may just be needed in the [ampnews](https://github.com/xwp/ampnews/) theme. It doesn't seem necessary in Twenty Fifteen. It's probably just that the ampnews theme lacks all of the required styles.